db/warehouse/migrate/20251027202455_add_automation_filters_to_cohorts.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddAutomationFiltersToCohorts < ActiveRecord::Migration[7.1]
+  def change
+    # use names that won't conflict with Cohort.available_columns
+    add_column :cohorts, :automation_sub_population, :string
+    add_column :cohorts, :automation_hoh_only, :boolean, default: false
```

Comment:
If we pivot to a `filter` json column, we could support additional filter options without needing to add columns to the table.  Even if for now we just set it to a has like: `{ automation_sub_population: :veteran, automation_hoh_only: false }` that feels like a good choice to me.   Am I missing something?

Reply:
It seemed faster to get the form inputs and model validations working using distinct columns. We could store them as json but then it adds some complexity.